sources/PTYTextView.m

```diff
@@ -5221,6 +5231,27 @@ - (void)setTransparency:(double)fVal
     [self setNeedsDisplay:YES];
 }
 
+- (void)setInactiveTransparency:(double)inactiveTransparency
+{
+    _inactiveTransparency = inactiveTransparency;
+    [_colorMap invalidateCache];
+    [self setNeedsDisplay:YES];
+}
+
+- (void)setUseInactiveTransparency:(BOOL)useInactiveTransparency
+{
+    _useInactiveTransparency = useInactiveTransparency;
+    [_colorMap invalidateCache];
+    [self setNeedsDisplay:YES];
+}
+
+- (void)setIsBackground:(BOOL)isBackground
```

Comment:
For consistency, this should be setIsInactive: and the backing variable should be _isInactive.